src/main/Footer.jsx

```diff
@@ -3,7 +3,15 @@ import React from "react";
 const Footer = () => {
   return (
     <div>
-      <p>This is fotter component</p>
+      <footer className="w-full text-center border-t border-grey pin-b">
+               <p className="flex justify-start pl-3 ">contact us</p>
+            <div className="flex justify-start bg-blue-400 text-white h-100px mt-100px">
```

Comment:
why are you using flex in both elements?
use the flex in their parent and then use `items-align` and `justify-content` to position the children
again, push all styling to css file and import it in the index